tests/ragger/test_pubkey_cmd.py

```diff
@@ -0,0 +1,16 @@
+def test_get_public_key(cmd):
+    pub_key, chain_code = cmd.get_public_key(
+        bip32_path="m/44'/0'/0'/0/0",
+        display=False
+    )  # type: bytes, bytes
+
+    assert len(pub_key) == 65
+    assert len(chain_code) == 32
+
+    pub_key2, chain_code2 = cmd.get_public_key(
+        bip32_path="m/44'/1'/0'/0/0",
+        display=False
+    )  # type: bytes, bytes
```

Comment:
We could also assert the values returned as the speculos test seed will always return the same values

Reply:
To be updated in another PR.